sklearn/utils/validation.py

```diff
@@ -1249,12 +1259,23 @@ def check_scalar(x, name, target_type, *, min_val=None, max_val=None):
         The minimum valid value the parameter can take. If None (default) it
         is implied that the parameter does not have a lower bound.
 
-    max_val : float or int, default=None
+    strictly_greater_min_val : bool, default=True
```

Comment:
or `strictly_gt` or `inclusive_gt`? It's kinda verbose right now

Reply:
yep this is a possibility as well

Reply:
I let the `max_val` `min_val` to be backward compatible thought.